tests/common.go

```diff
@@ -356,6 +362,48 @@ func ValidateAndDestroy(contexts []*scheduler.Context, opts map[string]bool) {
 	})
 }
 
+// AddLabelsOnNode adds labels on the node
+func AddLabelsOnNode(n node.Node, labels map[string]string) error {
+	for labelKey, labelValue := range labels {
+		if err := k8s.Instance().AddLabelOnNode(n.Name, labelKey, labelValue); err != nil {
```

Comment:
the tests directly should not assume k8s. This breaks the scheduler abstraction. Check for other places in PR too. 

Tests should invoke the scheduler driver interface APIs.